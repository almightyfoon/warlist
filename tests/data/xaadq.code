c9101b_-0L4t6r6r6U6U6U6U6U6U4x5U5UhVhV___-0khS6r4x4A4C6N6vn3n3cvgg5U6q6q6XhVhV
