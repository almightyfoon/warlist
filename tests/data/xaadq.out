https://conflictchamber.com/#c9101b_-0L4t6r6r6U6U6U6U6U6U4x5U5UhVhV___-0khS6r4x4A4C6N6vn3n3cvgg5U6q6q6XhVhV

Skorne Army - 75 / 75 points
[Theme] Imperial Warhost

[Mordikaar 1] Void Seer Mordikaar [+29]
 - Agonizer [0(6)]
 - Agonizer [0(6)]
 - Archidon [10]
 - Archidon [10]
 - Archidon [10]
 - Archidon [10]
 - Archidon [10]
 - Archidon [10]
 - Basilisk Krea [0(7)]
Paingiver Beast Handlers (min) [5]
Paingiver Beast Handlers (min) [5]
Siege Animantarax [17]
Siege Animantarax [17]

Skorne Army - 75 / 75 points
[Theme] Winds of Death

!!! This army contains pre-release entries.

[Morghoul 3] Dominar Morghoul & Escorts [+25]
 - Agonizer [6]
 - Basilisk Krea [7]
 - Cyclops Savage [7]
 - Cyclops Shaman [8]
 - Aptimus Marketh [0(5)]
Extoller Soulward [0(3)]
Immortal Vessel [3]
Immortal Vessel [3]
Mortitheurge Willbreaker [0(4)]
Venator Dakar [4]
Paingiver Beast Handlers (min) [5]
Venator Catapult Crew [5]
Venator Catapult Crew [5]
Venator Slingers (max) [13]
Siege Animantarax [17]
Siege Animantarax [17]
